src/annotator/sidebar.js

```diff
@@ -18,8 +18,16 @@ import BucketBar from './plugin/bucket-bar';
  * @prop {number} height
  */
 
+/**
+ * @typedef {{
+ *    eventTarget: Window| HTMLElement,
```

Comment:
There's an extra space before `HTMLElement` here, per our current conventions.

Reply:
Fixed.

Are the spaces here OK? `(event: any) => void` Or should I remove the spaces?